DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Repositories/TrustRepository.cs

```diff
@@ -78,6 +78,20 @@ public async Task<TrustGovernance> GetTrustGovernanceAsync(string uid)
         return governersDto;
     }
 
+    public async Task<TrustContacts> GetTrustContactsAsync(string uid)
+    {
+        var trm = await GetTrustRelationshipManagerLinkedTo(uid);
+        var sfso = await GetSfsoLeadLinkedTo(uid);
+        var governanceContacts = await GetGovernanceContactsAsync(uid);
+
+        return new TrustContacts(
+            trm,
+            sfso,
+            governanceContacts.GetValueOrDefault("Accounting Officer"),
+            governanceContacts.GetValueOrDefault("Chair of Trustees"),
```

Comment:
Consts for these values? they are used in multiple places

Reply:
log this as a technical debt ticket

Reply:
This will be completed as a separate piece of work [User Story 180233](https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/180233): Tech debt: Improve how we manage governance roles by using constant strings and enums